packages/commonwealth/server.ts

```diff
@@ -320,6 +321,9 @@ async function main() {
     rollbar
   );
 
+  // TODO: delete after this has run successfully in all envs
+  hexCreator.initJob(models, rollbar);
```

Comment:
Ready for code review, but waiting for a chance to test in Frick one more time...

Reply:
Frick is all yours!

Reply:
tested successfully, but I wonder if I should go back to a simple migration? Is 40k rows in a single transaction ok for Prod?